scripts/setup

```diff
@@ -46,9 +46,16 @@ dependency_setup \
 dependency_setup ./re-natal deps
 dependency_setup ./re-natal enable-source-maps
 
+if [[ -z "${ANDROID_NDK1}" ]]; then
+  dependency_setup \
+    "pushd android && ./gradlew react-native-android:installArchives && popd"
+else
+    cecho "@yellow[[Android NDK installation not detected. If you plan to build for Android, \
+please follow the instructions at https://docs.status.im/docs/build_status_linux.html to install the Android NDK:]]"
```

Comment:
How about if you build android on OSX?

Reply:
will add a check and display appropriate link